case_studies/get_redshift.py

```diff
@@ -0,0 +1,80 @@
+from astropy.io import fits
```

Comment:
@cczw123 I'm going to leave this file out for now. It's a bit messy and should be cleaned up before we add it to my branch; it's fine to just leave it for now as we're not pulling data from this source currently, we may use this in the future though.